integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -3172,48 +3172,6 @@ fn test_catchup_no_sharding_change() {
     }
 }
 
-/// Run `gc_num_epochs_to_keep` epochs + several blocks.
-/// Start a second env from the "snapshot" of the first.
-/// Run one more epoch.
-/// "Restart from the snapshot" is to ensure that we can continue producing blocks without relying on caches.
-#[test]
-fn test_long_chain_with_restart_from_snapshot() {
```

Comment:
Don't know why this test is failing but it has been superseded by testloop restarts tests.